Cargo.toml

```diff
@@ -12,32 +12,32 @@ exclude = [
 ]
 
 [dependencies]
```

Comment:
would be good to add `rust-version = "1.75"`

Reply:
can i do 1.79 ? since sdk uses it